lib/react_on_rails/server_rendering_pool.rb

```diff
@@ -48,6 +48,9 @@ def self.server_render_js_with_console_logging(js_code)
       result
     end
 
+    def self.before_render(component_name, props, prerender_options, request); ""; end
```

Comment:
I'm not clear on this. Is the idea to override/monkey-patch the ServerRenderingPool?